### ISSUE TITLE
Add a more descriptive output for the runs. 

### ISSUE THREAD

**Issue by rpkoller** [paulirish/pwmetrics#64]
At the moment if you are running an audit with more than one run you only get the generic line `Launching Chrome`: 

![bildschirmfoto 2017-02-26 um 14 49 01](https://cloud.githubusercontent.com/assets/1665422/23340366/1eeb465e-fc35-11e6-9bf8-c9b9977c228f.png)

In my opinion it would be more descriptive and helpful if the output would say something like `Run 1 out of 5 is finished successfully` and so on. That way you would also have the opportunity to deal with certain exceptions if something went wrong within the microcopy like `Unable to complete run 3 due to .....`

**Comment by pedro93** [comment]
@rpkoller @denar90 @paulirish if a run fails should it be re-run to ensure at least n runs successfully occur if a user specifies an average?

**Comment by rpkoller** [comment]
@pedro93 I would vote against the `Launching Chrome`message. That is the initial key of my suggestion to replace it with a more meaningful line like `Run 1 out of 5 is finished successfully`, so the user is aware that the first round of testing was finished successfully. `Launching Chrome`is a bit abstract and you see that anyway when the app focus switches from the terminal to Chrome. Therefor the `Run 1 out of 5 is finished successfully` is more reassuring. And in the case of an error you might introduce a default behaviour either just note it or rerun the test like you suggested. e.g. in case reruns are default you might write `pwmetrics http://example.com/ --runs=3 --no-rerun` as an override and you just get `Run 3 failed due to exception xyz`

**Comment by pedro93** [comment]
@paulirish @denar90 what do you think. I believe this is possible to do in the scope of the existing pull request. The re-run functionality I have to think about how to implement though given the current code structure.

**Comment by pedro93** [comment]
@rpkoller I understand your point of view but what about running pwmetrics in a CI system? It is helpful to know what stage of the process pwmetrics is in. I concede though that a static message does not give a lot of information

**Comment by rpkoller** [comment]
uh i haven't beared the CI case in mind. then you are totally right! sorry... But a suggestion for the output anyway: 

```
Run 1: Launching Chrome
Run 1: Run 1 of 3 finished successfully
```
That way it would be more aligned layout wise. and you could places the checkmark and the icon for failed processes easily prepended. and i would maybe add an empty line before the first `launching chrome` line and between the last run line and the  `*median run *` line so it looks a bit more cleaned up, loose and not that packed.

**Comment by pedro93** [comment]
@rpkoller I am afraid I don't understand what you mean by "could places the checkmark and the icon for failed processes easily prepended". Could you explain perhaps give an example?

**Comment by rpkoller** [comment]
@pedro93 slight typo ;) i meant "and you could place the check mark and the icon for failed processes" right before `Run 1: Launching Chrome` in each line. the checkmark like you already do with 

![bildschirmfoto 2017-03-09 um 02 14 16 2](https://cloud.githubusercontent.com/assets/1665422/23731598/e41cd58c-046e-11e7-8352-d0d4309ba7cb.png)

and a suggestion for failed processes like github uses: 

![bildschirmfoto 2017-03-09 um 02 14 16](https://cloud.githubusercontent.com/assets/1665422/23731595/e006fe8c-046e-11e7-9cc4-7fbfb2557779.png)

**Comment by pedro93** [comment]
If I understood correctly, then answer me this. 
The message ```Launching Chrome``` is sent to the console before pwmetrics calls lighthouse, it does not know beforehand if the requested process will finish correctly or not. 
How do you propose we prepend the checkmark/error icon before we know the result?

**Comment by rpkoller** [comment]
@pedro93 good and valid question. impossible so. ;) i was just brainstorming and looked at things from the usability perspective to provide something more descriptive and not from a code level. so forgot my suggestion in regards of the icons please ;)

**Comment by denar90** [comment]
merged #70

**Comment by denar90** [comment]
@pedro93 @rpkoller thanks guys!
